File: src/fizzBuzz.java
// Java FizzBuzz
public class fizzBuzz {
	public static void main(String[] args) {
		for (int i=1; i < 101; i++) {
			System.out.println(evaluate(i));
		}
	}
	public static String evaluate(int i) {
		if (i % 5 == 0 && i % 3 == 0) {
			return "FizzBuzz";
		}
		else if (i % 5 == 0) {
			return "Buzz";
		}
		else if (i % 3 == 0) {
			return "Fizz";
		}
		else {
			return String.valueOf(i);
		}
	}
}

File: src/fizzBuzz.py
#Python FizzBuzz
for x in range (1, 101):
    fizz = (x % 3 == 0)
    buzz = (x % 5 == 0)
    if fizz & buzz:
        print ('FizzBuzz')
    elif fizz:
        print ('Fizz')
    elif buzz: 
        print ('Buzz')
    else:
        print (x)
